canonical/canonical.com
# Pause Google Calendar API Usage

## Done
- This PR is to temporarily remove usage of the Google Calendar API on candidate withdrawal. The Calendar API was being used to automatically delete interview events when a candidate withdraws. However, the API key is not working in production and is causing errors. As a result, to ensure that candidates can still withdraw while this issue is being debugged, this PR removes the Calendar API usage entirely for the time being.
- To do this, we comment out all of the Google Calender-related lines in `application.py`
- We also remove tests involving the google calendar API so that they won't error

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8002/
